tests/plugin/data/sw_loguru/services/provider.py

```diff
@@ -24,21 +25,35 @@
 
     app = FastAPI()
 
+    logging_logger = logging.getLogger()
+
 
     @app.get('/users')
     async def application():
         time.sleep(0.5)
 
         try:
             raise Exception('Loguru Exception Test.')
-        except Exception:   # noqa
+        except Exception:  # noqa
             logger.opt(exception=True).error('Loguru provider error reported.')
+            time.sleep(0.5)
+            logging_logger.error('Logging provider error reported.', exc_info=True)
+
+        time.sleep(0.5)
 
+        # this will be filtered by SW_AGENT_LOG_REPORTER_LEVEL
         logger.debug('Loguru provider debug reported.')
 
+        time.sleep(0.5)
+
         logger.warning('Loguru provider warning reported.')
 
+        time.sleep(0.5)
+
```

Comment:
Can we remove these `time.sleep`?

Reply:
> Can we remove these `time.sleep`?

Of course. I referred other test file.

Reply:
I think the sleep was added because of this bug https://github.com/apache/skywalking/issues/8752 that introduced flaky tests in e2e, but it shouldn't be a problem for you in plugin tests, so can be safely removed.